mllib-local/pom.xml

```diff
@@ -61,6 +61,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Oh, I don't think we want to introduce this circular dependency. What's it for?

Reply:
for Utils.classForName

Reply:
We can't do it for that. Just inline what you need. There may be a 'copy' in an MLlib utils class even, not sure

Reply:
> We can't do it for that. Just inline what you need. There may be a 'copy' in an MLlib utils class even, not sure

+1, Agree